Step04/app/src/main/res/drawable-v21/ic_menu_send.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillColor="#FF000000"
+        android:pathData="M2.01,21L23,12 2.01,3 2,10l15,2 -15,2z" />
+</vector>
```

Comment:
Let me hold this for now. Because after I renamed it, I have faced unexpected errors which I couldn't solve with Google's help. Looks like its android studio bug.